src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java

```diff
@@ -135,6 +136,19 @@ public CompletableFuture<TransactionResult> multipleTransaction(
         return account;
     }
 
+	public boolean deleteBankAccount(@NotNull String name) {
```

Comment:
Could we have a interface method for this please?
https://github.com/EconToolBox/AccountInterface/blob/main/src/main/java/org/kaiaccount/account/inter/type/named/bank/BankAccount.java